### ISSUE TITLE
Add PostCss

### ISSUE THREAD

**Issue by benjaminstrasser** [codingcommons/tiny-tms#78]
We are getting the following warning inside of #74  and #77 

```
/home/bstrasser/Documents/projects/open-source/tiny-tms/src/components/ui/theme-selector/theme-selector.svelte:56:3
Warn: Unknown at rule @apply (css)
        input[type='radio']:checked + label {
                @apply border border-primary;
        }


/home/bstrasser/Documents/projects/open-source/tiny-tms/src/routes/(authenticated)/+layout.svelte:45:3
Warn: Unknown at rule @apply (css)
        .selected {
                @apply bg-primary-foreground;
        }
```
Adding postcss should fix it

https://github.com/sveltejs/language-tools/blob/5a6eba15918a556e2bcb04638c313ac9dd03212c/docs/preprocessors/other-css-preprocessors.md

**Comment by sjaghori** [comment]
We use vitePreprocess and it uses postcss as default. This might be a different issue

**Comment by benjaminstrasser** [comment]
> We use vitePreprocess and it uses postcss as default. This might be a different issue

You were right. Fixed in 472bf508e0df9621dc8f9c87a75fe4d4f560553f
